distro/scenarios/theNewWorld/culture.properties

```diff
@@ -339,7 +339,6 @@
 1673,1354,FARME
 696,350,HUNTE
 2023,1812,FARME
-2272,1736,HUNTE

```

Comment:
Hmmm, that's weird... I don't think we should move them because they were based on real life data